frontend/src/components/Navbar/Navbar.js

```diff
@@ -4,22 +4,30 @@ import "./Navbar.css";
 
 import logo from "../../res/logo_simple.svg";
 
-function Navbar() {
+function Navbar(props) {
+  const hiddenAdminFilter = keyValuePair => {
+    let pageObject = keyValuePair[1];
+    if (pageObject.hidden) return false;
+    if (!pageObject.admin) return true;
+    if (pageObject.admin && props.admin) return true;
+    return false;
```

Comment:
could we simplify this?
```suggestion
    let pageObject = keyValuePair[1];
    if (pageObject.hidden) return false;
    if (pageObject.admin && !props.admin) return false;
    return true;
```
maybe 🤔

Reply:
Now done in https://github.com/amberstarlight/cutieclubsite/pull/13/commits/5627fc73e187751e8621f9bc2adc48cfb8230550 :smile: